shared/applicability/oval/installed_app_is_eks.xml

```diff
@@ -4,7 +4,8 @@
     <metadata>
       <title>Amazon Elastic Kubernetes Service</title>
       <affected family="unix">
-        <platform>Amazon Elastic Kubernetes Service</platform>
+        <platform>multi_platform_eks</platform>
+        <platform>multi_platform_ocp</platform>
```

Comment:
Why was `multi_platform_ocp` added here?
Is this required for the content to build?

Reply:
Yes. The longer answer is: we technically need both these checks (it is a CPE check) in the DS to tell platforms apart.

Reply:
It is symmetrical to this problem: https://github.com/ComplianceAsCode/content/pull/10610

Reply:
Better yet: https://github.com/ComplianceAsCode/content/pull/11436

Reply:
I see, thanks.